src/pytest_inmanta_lsm/orchestrator_container.py

```diff
@@ -32,16 +32,20 @@ def run_cmd(*, cmd: List[str], cwd: Path) -> Tuple[str, str]:
     LOGGER.info(f"Running command: {cmd}")
     env_vars = dict(os.environ)
     env_vars.pop("PYTHONPATH", None)
-    result = subprocess.run(
-        args=cmd,
-        cwd=str(cwd),
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        encoding="utf-8",
-        text=True,
-        universal_newlines=True,
-        env=env_vars,
-    )
+    try:
+        result = subprocess.run(
+            args=cmd,
+            cwd=str(cwd),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            encoding="utf-8",
+            text=True,
+            universal_newlines=True,
+            env=env_vars,
+        )
+    except FileNotFoundError:
+        raise FileNotFoundError("The `docker-compose` command is not found. You need it to have a local orchestrator.")
```

Comment:
I would add an additional check that the missing file is indeed `docker-compose`, because it could be also something else (like the `docker-compose.yml` file)